test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/Asn1TestData.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging.Test
+{
+    internal static class Asn1TestData
+    {
+        internal static byte[] SigningCertificate
```

Comment:
Maybe a short description of what properties this certificate has and how it was generated.